content/docs/user-guide/checkpoints.md

```diff
@@ -0,0 +1,386 @@
+│ │ ╟ 4067622           │ 01:41 PM     │   17 │ 0.050389 │ 0.9823 │ 473987 │ 0.001 │ 0            │
+│ │ ╟ 05a0883           │ 01:41 PM     │   16 │ 0.065079 │ 0.9808 │ 473987 │ 0.001 │ 0            │
+│ │ ╟ c23a6e0           │ 01:41 PM     │   15 │  0.05095 │ 0.9831 │ 473987 │ 0.001 │ 0            │
+│ │ ╟ a8ed9aa           │ 01:40 PM     │   14 │ 0.056475 │ 0.9819 │ 473987 │ 0.001 │ 0            │
+│ │ ╟ 0b5c7f0           │ 01:40 PM     │   13 │ 0.054636 │ 0.9817 │ 473987 │ 0.001 │ 0            │
+│ │ ╟ 6c4523d           │ 01:40 PM     │   12 │ 0.060063 │  0.982 │ 473987 │ 0.001 │ 0            │
+│ │ ╟ 54bca46           │ 01:40 PM     │   11 │  0.07912 │ 0.9747 │ 473987 │ 0.001 │ 0            │
+│ │ ╟ c60fe61 (574bdc3) │ 01:40 PM     │   10 │ 0.082944 │ 0.9728 │ 473987 │ 0.001 │ 0            │
+│ │ ╓ exp-e81d6         │ 01:30 PM     │    9 │ 0.066135 │ 0.9792 │ 473987 │ 0.001 │ 0            │
+│ │ ╟ 34b1d6f           │ 01:30 PM     │    8 │ 0.066629 │ 0.9783 │ 473987 │ 0.001 │ 0            │
+│ │ ╟ c048966           │ 01:30 PM     │    7 │ 0.082436 │ 0.9747 │ 473987 │ 0.001 │ 0            │
+│ │ ╟ 2910f35           │ 01:30 PM     │    6 │  0.08161 │ 0.9749 │ 473987 │ 0.001 │ 0            │
+│ │ ╟ 255d3aa           │ 01:30 PM     │    5 │ 0.088654 │ 0.9716 │ 473987 │ 0.001 │ 0            │
+```
+
+The existing checkpoint is referenced at the beginning of the new experiment.
+The new experiment is referred to as a modified experiment because it's taking
+existing data and using it as the starting point.
+
+### Metrics diff
+
+When you've run all the experiments you want to and you are ready to compare
+metrics between checkpoints, you can run the command: `dvc metrics diff`. You'll
+see something similart to this in your terminal.
+
+```dvc
+Path          Metric    Old    New      Change
+dvclive.json  acc       —      0.9854   —
+dvclive.json  loss      —      0.04828  —
+dvclive.json  step      —      19       —
+```
+
+### Looking at plots
+
+You also have the option to generate plots to visualize the metrics about your
+training epochs. Running `dvc plots show` will generate a `plots.html` file that
+you can open in a browser and it will display plots for you similar to the ones
+below.
+
+![Plots generated from running experiments on MNIST dataset using DVC](/img/checkpoints_plots.png)
+
+## Adding checkpoints to your Git workspace
+
+When the final epoch has run, you'll see a few commands in the terminal that
+will allow you to add these changes to Git.
+
+```dvc
+To track the changes with git, run:
+
+        git add dvclive.json dvc.yaml .gitignore train.py dvc.lock
+
+Reproduced experiment(s): exp-263da
+Experiment results have been applied to your workspace.
+
+To promote an experiment to a Git branch run:
+
+        dvc exp branch <exp>
+```
+
+You could also run the following command to promote your experiments to the Git
+workspace.
+
+```bash
+dvc exp apply exp-263da
+git add .
+git commit -m 'optimize model'
+```
```

Comment:
Do we need `apply` here? I see two possible actions (from the output):
1. `git add dvclive.json dvc.yaml .gitignore train.py dvc.lock && git commit -m 'my experiment'` to make a single commit in the current branch (like master).
2. `dvc exp branch <exp>` to create a branch that can be stored for later or pushed to GitHub/CiCd.

The first seems like an easy option and we can provide a code sample for a user to run.